MapboxAndroidDemo/src/main/java/com/mapbox/mapboxandroiddemo/MainActivity.java

```diff
@@ -136,6 +160,16 @@ public void onItemClicked(RecyclerView recyclerView, int position, View view) {
       navigationView.setNavigationItemSelectedListener(this);
       navigationView.setCheckedItem(R.id.nav_basics);
     }
+
+    loggedIn = PreferenceManager.getDefaultSharedPreferences(getApplicationContext())
+      .getBoolean("TOKEN_SAVED", false);
```

Comment:
Ok, I've added it to a new `SharedPrefConstants` class as 
`public static final String TOKEN_SAVED_KEY = "TOKEN_SAVED";`